spec/companies_house/client_spec.rb

```diff
@@ -195,4 +195,40 @@
       end
     end
   end
+
+  describe "#company_search" do
+    include_context "test client"
+
+    subject(:request) { client.company_search(query, limit) }
```

Comment:
It might be nicer to call this `response`, since the result is a parsed HTTP response body.